pkg/controllers/dynakube/processmoduleconfigsecret/reconciler.go

```diff
@@ -30,120 +30,95 @@ type Reconciler struct {
 	dtClient     dtclient.Client
 	dk           *dynakube.DynaKube
 	timeProvider *timeprovider.Provider
+	secretQuery  k8ssecret.QueryObject
```

Comment:
Do we want to start adding query types into reconciler structs? I'm fine with it, just asking because we don't do it in other places.

Reply:
just my 2 cents: I wouldn't mind adding it as well, I would just like to keep it consistent and add the query to the struct on all other locations.

Reply:
Changed it back - Will create a small ticket to edit it everywhere, don't want to do it in this PR though.